Sources/Apodini/Semantic Model Builder/SharedSemanticModelBuilder.swift

```diff
@@ -38,10 +38,10 @@ class WebServiceModel {
 }
 
 class SharedSemanticModelBuilder: SemanticModelBuilder, InterfaceExporterVisitor {
-    private var interfaceExporters: [AnyInterfaceExporter] = []
+    var interfaceExporters: [AnyInterfaceExporter] = []
```

Comment:
Is there a reason for removing the private modifier? I couldn't find one in the code.

Reply:
Thanks for pointing this out. No, there is no reason (anymore) 👍